### PR TITLE
tools/cephfs-shell:allow to unittest shell using transcript

### PR DESCRIPTION
**Pull request by rpavani1998**
Allow to test the shell using transcript.
Signed-off-by: Pavani Rajula <rpavani1998@gmail.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/tools/cephfs/cephfs-shell [hunk 1]
```diff
@@ -475,7 +475,7 @@ class CephFSShell(Cmd):
                 elif args.long:
                     print_long(self, cephfs.getcwd().decode('utf-8') + dir_name + '/' + path, flag, False)
                 else:
-                    values.append(colorama.Fore.BLUE * flag + path + '/'* flag + colorama.Style.RESET_ALL  * flag)
+                    values.append(self.colorize(path + '/'* flag, 'blue'))
```
Comment (tchaikov) [inline src/tools/cephfs/cephfs-shell]: might want to put
```py
elif flag:
  values.append(self.colorize(path + '/'))
else:
  values.append(path)
```

Comment (tchaikov) [reply src/tools/cephfs/cephfs-shell]: BTW, `flag` deserves a better name, IMHO.

**Comment by tchaikov** [comment]
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

**Comment by rpavani1998** [comment]
@tchaikov updated.

File: src/tools/cephfs/cephfs-shell [hunk 2]
```diff
@@ -457,7 +457,7 @@ class CephFSShell(Cmd):
                 items = sorted(list_items(dir_name), key = lambda item: item.d_name)
             if not args.all and len(items) > 2:
                 items = [i for i in items if not i.d_name.decode('utf-8').startswith('.')]
-            flag = 0
+            is_dir = 0
```
Comment (tchaikov) [inline src/tools/cephfs/cephfs-shell]: how about make it a boolean?

File: src/tools/cephfs/cephfs-shell [hunk 3]
```diff
@@ -957,10 +957,12 @@ if __name__ == '__main__':
...
+    if args.test:
+        args.commands = [arg+',' for arg in sys.argv[sys.argv.index('-t'):]]
     sys.argv.clear()
     sys.argv.append(exe)
-    sys.argv.extend(' '.join(args.commands).split(','))
+    sys.argv.extend([i.strip() for i in ' '.join(args.commands).split(',')])
     setup_cephfs(config_file)
     c = CephFSShell()
-    c.cmdloop()
+    c.cmdloop()
```
Comment (batrick) [inline src/tools/cephfs/cephfs-shell]: Your editor seems to be removing the final newline in the file.

**Comment by batrick** [comment]
@rpavani1998 you still have the colorize commit in this PR :)

**Comment by rpavani1998** [comment]
@batrick updated

**Pull request merged**